### ISSUE TITLE
URL bar suggestions can only be navigated using down key when suggestion list is shown

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#7682]
## Test plan

----

- Did you search for similar issues before submitting this one?
Yes

- Describe the issue you encountered:
URL bar suggestions can only be navigated using down key when suggestion list is shown

- Platform (Win7, 8, 10? macOS? Linux distro?):
All

- Brave Version (revision SHA):
Brave	0.13.5
rev	1db81cb

- Steps to reproduce:
    1. Ensure all suggestions switch is enabled in the search setting page
    2. Have sufficient history so that suggestion are shown based on all the options
    3. Try using UP arrow key to select the bottom most suggestion list, doesn't work. Have to scroll down or use a mouse to open the suggested item
    
- Actual result:
Press UP arrow doesn't select the last suggestion 

- Expected result:
Navigation should be both ways using UP and Down arrow key on URL bar suggestions

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
Yes

- Is this an issue in the currently released version?
Yes

- Can this issue be consistently reproduced?
Yes

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:
![downarrow](https://cloud.githubusercontent.com/assets/17010094/23865063/506b7176-080c-11e7-8c97-a4e22e2aaacc.gif)

- Any related issues:
